enos/enos-variables.hcl

```diff
@@ -33,7 +33,7 @@ variable "artifactory_repo" {
 variable "aws_region" {
   description = "The AWS region where we'll create infrastructure"
   type        = string
-  default     = "us-west-1"
+  default     = "us-east-2"
```

Comment:
Is there a particular reason we change the default region?

Reply:
Good catch! I had changed it when trying to setup replication between different zones, but didn't change it back. Will update it now.